coffeelint.json

```diff
@@ -55,7 +55,7 @@
   },
   "newlines_after_classes": {
     "value": 1,
-    "level": "error"
+    "level": "ignore"
```

Comment:
According to http://www.coffeelint.org/ this rule is ignore by default. Not sure why we had it marked as an error. We have errors on this cop in other files, and it seems to me to be a weird one (especially the 1 value). It would mean that the `$ ->` line would come directly after the last line of the class.

Reply:
👍